2682. Find the Losers of the Circular Game/solution.py

```diff
@@ -0,0 +1,29 @@
+class Solution:
+    # taken the number of players, n, and a constant, k, it returns the losers of a circular game. player 1 starts with the ball,
+    # and passes it to the person k times away from him. The same thing for 2*k. Then 3*k. And so on until the player who has 
+    # received the ball 2 times wins and the players who didn't receive the ball at all lose.
+    def circularGameLosers(self, n: int, k: int) -> List[int]:
+        # define an array of the people who will currently lose
+        losers = []
+        for i in range(n):
+            losers.append(i + 1)
+        print(losers)
+        # which player holds the ball?
+        ballHolder = 1
+        # how many iterations have gone by? (it determines who gets the ball after each round)
+        iterations = 0
+        # simulate the game
+        while True:
+            # remove the current ballholder from the losers list. if the ballholder has already received the ball once, or is 
+            # not a potential loser, we return all the losers. 
```

Comment:
Just commenting that this is a different approach from my code. I would simply determine where the ball would go next, and if a previous ball owner got it, then I would terminate the loop. Otherwise, I would keep iterating.